lib/tasks/test_track_rails_client_tasks.rake

```diff
@@ -31,3 +42,4 @@ end
 task 'db:schema:load' => ['test_track:schema:load']
 task 'db:structure:load' => ['test_track:schema:load']
 task 'db:migrate' => ['test_track:migrate']
+task 'deploy:assets:precompile' => ['test_track:generate_build_timestamp']
```

Comment:
Would be nice to add a test for this since it’s now pretty critical to building the apps, not sure if we have any examples of testing rake tasks.